### ISSUE TITLE
dependency warning

### ISSUE THREAD

**Issue by zhenyulin** [styled-components/stylelint-processor-styled-components#141]
I'm currently getting the following warning:
```
warning "stylelint-processor-styled-components > typescript-eslint-parser@7.0.0" has unmet peer dependency "typescript@*".

**Comment by emilgoldsmith** [comment]
Sorry for the nuisance, see this comment: https://github.com/styled-components/stylelint-processor-styled-components/issues/12#issuecomment-341922169

**Comment by zhenyulin** [comment]
@emilgoldsmith Thanks for the quick followup. I'm currently on v1.0.0, but still get this warning.

**Comment by emilgoldsmith** [comment]
Then you didn't read the comment I linked to :). Maybe you only read the one above it? Read both of my comments :)

**Comment by zhenyulin** [comment]
@emilgoldsmith OK, thanks for the clarification. Was wondering if it is proper to treat it as a devDepedency?

**Comment by emilgoldsmith** [comment]
I'm sorry @zhenyulin but you're tiring me out a bit now as it seems like you keep on not reading my comments, this is the second time in a row you ask something I already addressed in the comment I linked to you.

**Comment by zhenyulin** [comment]
@emilgoldsmith sorry, I've taken it wrong earlier. Found it on one of the earlier comments above.
